src/components/AppsList/index.jsx

```diff
@@ -101,6 +97,7 @@ class AppsList extends Component {
     );
     return (
       <div >
+        {console.log(allApps)}
```

Comment:
is there any reason why you loging this?

Reply:
nice cache @jnsereko